Makefile

```diff
@@ -50,7 +50,7 @@ vet: ## Run go vet against code.
 	go vet ./...
 
 test: manifests generate fmt vet envtest ## Run tests.
-	go test -coverpkg=./... -coverprofile=cover.out --timeout=20m ./...
+	go test $(go list ./...  | grep -v ./emqx-operator/apis/apps/v1beta3/zz_generated.deepcopy.go) -coverprofile=cover.out --timeout=20m ./...
```

Comment:
I don't think that's a good way to go, we'll have many apiVersions

Reply:
already rolled back